workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-search-category-facet-tabs/ddm-template.ftl

```diff
@@ -42,15 +42,30 @@
 </#if>
 
 <@liferay_aui.script>
-	function ${namespace}updateSelection(event) {
-		const form = event.currentTarget.form;
+	  function handleStyleTabs(event) {
```

Comment:
@luisacmn make sure we are always using tabs instead of spaces. You should be able to set your editor to show spaces, so its more evident, but also the formatter should catch it for you if you run it